components_beta/template/my_component/frontend/src/ReactTemplate.tsx

```diff
@@ -8,7 +8,6 @@ import {
 // We import bootstrap.css and streamlit.css to get some simple default
 // styling for our text and button. You can remove or replace these!
 import "bootstrap/dist/css/bootstrap.min.css"
-import "./streamlit.css"
 
```

Comment:
Do we still need bootstrap, or does the streamlit CSS handle margins and borders and good default text and widget styles now?